modules/creating-ibm-power-vs-workspace-procedure.adoc

```diff
@@ -10,10 +10,10 @@ Use the following procedure to create an {ibmpowerProductName} Virtual Server wo
 
 . To create an {ibmpowerProductName} Virtual Server workspace, complete step 1 to step 5 from the IBM Cloud documentation for link:https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server[Creating an IBM Power Virtual Server].
 
-. After it has finished provisioning, retrieve the 32-character alphanumeric ID of your new workspace by entering the following command:
+. After it has finished provisioning, retrieve the 32-character alphanumeric ID of your new workspace mentioned in front of the GUID field of the output of the following command:
 +
 [source,terminal]
 ----
-$ ibmcloud resource service-instances | grep <workspace name>
+$ ibmcloud resource service-instances <workspace name>
```

Comment:
`ibmcloud resource service-instance <workspace name>`
`service-instance` is singular in the command to retrieve details of a single workspace.

Reply:
Updated.